src/storage/constants.rs

```diff
@@ -41,6 +41,8 @@ pub enum RootPrefix {
 
     /* Used to maintain information about latest onchain events, fnames ingested */
     NodeLocalState = 17,
+
+    Proposal = 18,
```

Comment:
Hmm, I wonder if this should go under NodeLocalState (and use the node local db instead of the shard dbs). Mainly because this is specific to the proposer. e.g. You don't want this info included in the snapshot. It doesn't hurt because it's unlikely it will matter when resuming from snapshot, but it's cleaner to not have to worry about this kind of issue in the future.

Reply:
Yeah, I think it makes sense to put in `NodeLocalState`. Moved.